radio/src/gui/taranis/menu_general_setup.cpp

```diff
@@ -473,9 +477,9 @@ void menuGeneralSetup(uint8_t event)
         if (attr && s_editMode>0) {
           CHECK_INCDEC_GENVAR(event, g_eeGeneral.stickMode, 0, 3);
```

Comment:
I remember one issue we solved in the past with this code, somebody going from mode1 to mode3 with the model ON. Perhaps @romoloman will remember the details, I discussed this with him at the time